python/cudf/cudf/core/groupby/groupby.py

```diff
@@ -618,10 +618,9 @@ def __init__(
             For aggregated output, return object with group labels as
             the index. Only relevant for DataFrame input.
             as_index=False is effectively “SQL-style” grouped output.
-        sort : bool, default True
-            Sort group keys. Get better performance by turning this off.
-            Note this does not influence the order of observations within each
-            group. Groupby preserves the order of rows within each group.
```

Comment:
This seems like a plain rephrase from pandas doc - which seems to contradict to what it says above. 

A separate question, if currently cudf does not support `groupby.transform`, is this line relevant in cudf context?

Reply:
This is correct as we now use a stable sort for sort based groupbys in the case of certain non-aggregation functions, i.e. `groupby.apply`